.storybook/themes/news.scss

```diff
@@ -812,6 +812,53 @@
 					list-style-position: inside,
 					list-style-type: disc,
 				),
+				author-bio: (
+					gap: var(--global-spacing-4),
+				),
+				author-bio-author: (
+					gap: var(--global-spacing-4),
+					display: flex,
+					flex-direction: row,
+					components: (
+						image: (
+							max-width: var(--global-spacing-12),
+							max-height: var(--global-spacing-12),
+							border-radius: var(--global-border-radius-2),
+						),
+						stack: (
+							flex: 1,
+						),
+					),
+				),
+				author-bio-social-link-wrapper: (
+					display: flex,
+					flex-wrap: wrap,
+					components: (
+						link: (
+							margin: var(--global-spacing-2) var(--global-spacing-4) 0 0,
```

Comment:
Set `gap: var(--global-spacing-4) var(--global-spacing-3)` on the `-wrapper`, you don't need these margins, it would align with the design, will not make the links too large, and not possibly cause a violation for being too close on mobile.

Reply:
@haganka  confirmed that the gap between social-icons is 16px.

Reply:
sorry, yes.  I read the wrong dimension.  It would be: `gap: var(--global-spacing-4)` above instead of the margins to help alleviate the links bumping each other.